packages/server/api/controllers/favorites.controller.js

```diff
@@ -0,0 +1,73 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+// get
+const getFavorites = async () => {
+  try {
+    const favorites = await knex.select().table('favorites');
```

Comment:
Shouldn't this be scoped per user? Otherwise, everyone has access to all other user's favorites.

Reply:
I dont get it How to change it ?

Reply:
Its done now !